fpmsyncd/routesync.cpp

```diff
@@ -897,6 +1004,123 @@ bool RouteSync::getIfName(int if_index, char *if_name, size_t name_len)
     return true;
 }
 
+#ifndef LIBNL3_ENCAP_MPLS
```

Comment:
lets remove this and update azure pipeline to fetch libnl3 debian package from sonic-buildimaage

Reply:
@smaheshm I am not familiar with the azure pipeline infrastructure and will need input with exact changes required to fetch libnl3 debian  packages from sonic-buildimage.

Reply:
@smaheshm I have submitted a new PR https://github.com/Azure/sonic-buildimage/pull/8064 that is the only other change required to make this PR work in the Azure pipeline as it is currently.